docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -208,15 +230,53 @@ The API returns the following response:
   "timed_out": false,
   "num_reduce_phases": 4,
   "_shards": {
-    "total": 3,
-    "successful": 3,
+    "total": 12,
+    "successful": 12,
     "failed": 0,
     "skipped": 0
   },
   "_clusters": {
     "total": 3,
     "successful": 3,
-    "skipped": 0
+    "skipped": 0,
+    "details": {
+      "(local)": {            <1>
+        "status": "successful",
```

Comment:
quick q: there is no way to represent this as http status code? I was looking at this and being surprised with some new type of info being returned and possibly not knowing what the options are and how to interpret it. it is though quite self-explanatory. Do we have other places where we return similar information in other API?

Reply:
HTTP status code wouldn't work since PARTIAL and SUCCESSFUL are both "successful" (200 OK) for clusters (similar to how for shards "skipped" and "successful" are "success" statuses). And what HTTP status would we use for "skipped"? I guess we could use 400, but the overall query is still a success (200 OK). I think that would fairly cryptic to end users.

The status value here is meant to refer to the cluster "accounting" options above: total/successful/skipped. I plan to add "failed" to the accounting list at some point, but left it out of this PR since it is getting large.

So to address the concern of "I don't know what all possible states are", we could list those in the counter section:
```
  "_clusters": {
    "total": 5,
    "successful": 2,
    "partial": 1,    // not present
    "skipped": 1,
    "failed": 0,     // plan to add later
    "details": {
      "(local)": {           
        "status": "successful",
         ...
       }
      "remote1": {           
        "status": "skipped",
```

Reply:
Thanks for expanding. The proposed change makes sense to me, as a lower priority follow-up.